openstudiocore/src/utilities/units/Scale.cpp

```diff
@@ -114,6 +114,20 @@ namespace openstudio{
     static Scale M = {"M", "mega",6,1.0E6};
     return M;
   }
+  
+  /** Static constant defining SI prefix for 10^5. */
+  UTILITIES_API const Scale& hectokilo()
+  {
+    static Scale hk = {"_hk", "hectokilo",5,1.0E5};
+    return hk;
+  }
+
+  /** Static constant defining SI prefix for 10^4. */
```

Comment:
Ah ok I missed that k was just a local variable.  I don't think we need to put in something for 1E4 if the [official NIST guide to SI](http://www.nist.gov/pml/pubs/sp811/sec04.cfm) doesn't even have one.